### PR TITLE
Add pipelines aa1 to implement 1

### PR DESCRIPTION
**Pull request by Aichnerc**
Pipelines including a search feature have been implemented.

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no pipelines page.

**What is the new behavior (if this is a feature change)?**
- The pipelines page including search functionality has been added.

### CODE REVIEW

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by pinterid — APPROVED** [review]
Looks good.

File: src/index.js [hunk 1]
```diff
@@ -45,6 +49,13 @@ import registerServiceWorker from "./registerServiceWorker";
 //  return INTEL;
 //};
 
+// Initialize time ago
+JavascriptTimeAgo.addLocale(en);
+
+const getIntel = () => {
+  return true;
+};
+
```
Comment (schettn) [inline src/index.js:57]: Intel will be not required in [ops-front](https://github.com/snek-at/ops-front).

File: src/index.js [hunk 2]
```diff
@@ -55,7 +66,13 @@ const composeEnhancers =
     : compose;
 
 const enhancer = composeEnhancers(
-  applyMiddleware(loadingBarMiddleware())
+  applyMiddleware(
+    loadingBarMiddleware(),
+    thunk.withExtraArgument({
+      // Intel
+      getIntel,
```
Comment (schettn) [inline src/index.js:72]: Remove this
```suggestion
```

File: src/index.js (hunk 1 above)
Comment (schettn) [reply src/index.js:57]:
```suggestion
```

File: src/components/organisms/sections/Pipelines/index.jsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  MDBListGroup,
+  MDBListGroupItem,
+  MDBIcon,
+  MDBInput,
+} from "mdbreact";
+//> Additional
+import ReactTimeAgo from "react-time-ago";
+//> Actions
+// Functions to send data from the application to the store
+import { getPipelines } from "../../../../store/actions/pipelineActions";
```
Comment (schettn) [inline src/components/organisms/sections/Pipelines/index.jsx:23]:
```suggestion

//> Actions
// Functions to send data from the application to the store
import { getPipelines } from "../../../../store/actions/pipelineActions";
```

File: src/index.js [hunk 4]
```diff
@@ -24,6 +24,10 @@ import "bootstrap-css-only/css/bootstrap.min.css";
 // "Material Design for Bootstrap" is a great UI design framework
 import "mdbreact/dist/css/mdb.css";
 
+//> React Time Ago
+import JavascriptTimeAgo from "javascript-time-ago";
+import en from "javascript-time-ago/locale/en";
```
Comment (schettn) [inline src/index.js]:
```suggestion
// "Material Design for Bootstrap" is a great UI design framework
import "mdbreact/dist/css/mdb.css";
//> React Time Ago
import JavascriptTimeAgo from "javascript-time-ago";
import en from "javascript-time-ago/locale/en";
```

**Review by schettn — CHANGES_REQUESTED** [review]
Have a look

File: src/index.js (hunk 1 above)
Comment (Aichnerc) [reply src/index.js:57]: This is only a placeholder so this wonderful app does not crash on mi boi.

File: src/index.js (hunk 2 above)
Comment (Aichnerc) [reply src/index.js:72]: No u.

**Review by schettn — APPROVED** [review]
+1

**Pull request merged**